svc/TrackService.js

```diff
@@ -10,8 +10,27 @@ import {stripTags} from 'hoist/utils/HtmlUtils';
 
 export class TrackService extends BaseService {
 
+    /**
+     * Create a Track Log entry.
+     * Client metadata is set automatically by the server's parsing of request headers.
+     *
+     * @param {Object|string} options - if a string, it will become the msg value.
+     * @param {string} [options.msg] - user-supplied message. Required if options is an object.
+     * @param {string} [options.message] - same as options.msg.  Just supporting msg & message. Use either, not both.
+     * @param {string} [options.category] - user-supplied category.
+     * @param {Object|Array} [options.data] - user-supplied data collection.
+     * @param {number} [options.elapsed] - time in milliseconds some activity took.
+     * @param {string} [options.severity] - importance flag, such as: OK|WARN|EMERGENCY
+     *                 (errors should be tracked by the ErrorTrackingService, not sent
+     *                 in this TrackService).
```

Comment:
I'm sorry really not trying to drive everyone crazy, but we've been going with the standard where param docs that need to wrap to multiple lines just get indented three tabs (twelve spaces) from the left margin, so they end up under the "m" in param.

That way if you have multiple param comments that wrap they don't end up with different left-edges due to the potentially variable length of the {type} declarations.  And in this case then you can end up with just two lines overall for this param.

Reply:
fixed.